backend/server.py

```diff
@@ -226,43 +235,49 @@ def add_status():
     status_data = {}
     status_data['nodes'] = []
 
-
     return jsonify(return_data)
 
 @app.route('/fetcher/statistics', methods=['GET', 'POST'])
 def add_statistics():
+    timeout=default_values.timeout_statistics

```

Comment:
Could we use `default_values.timeout_statistics` directly in the wait method call instead of assigning it to a separate `timeout` variable? This change might simplify the code and make it more concise.